packages/tables/src/Testing/TestsRecords.php

```diff
@@ -27,7 +27,7 @@ function ($record) use ($livewire) {
                         $record = $livewire->getTableRecordKey($record);
                     }
 
-                    return "wire:key=\"table.records.{$record}\"";
+                    return "{$livewire->id}.table.records.{$record}";
```

Comment:
@danharrin isn't this wrong?, where is wire:key=xxxx

Reply:
No this isn't an issue, I want to decouple the tests from this specific attribute in the view. It still works fine for you, right?

Reply:
Didn't have a chance to test,..
Just thought it was a mistypo to remove wirekey.